### ISSUE TITLE
check failed: git: invalid repository: unsecured

### ISSUE THREAD

**Issue by xmlking** [rvflash/goup#1]
for my [repo](https://github.com/xmlking/logger) I am getting following error 

check failed: git: invalid repository: unsecured call to http://../ cancelled: failed to list tags

```
goup -v -m ./...

goup: github.com/xmlking/logger: github.com/pkg/errors v0.9.1 is up to date
goup: github.com/xmlking/logger: github.com/rs/zerolog v1.18.0 is up to date
goup: github.com/xmlking/logger: github.com/sirupsen/logrus v1.4.2 is up to date
goup: github.com/xmlking/logger/gormlog: ../ check failed: git: invalid repository: unsecured call to http://../ cancelled: failed to list tags
goup: github.com/xmlking/logger/logrus: ../ check failed: git: invalid repository: unsecured call to http://../ cancelled: failed to list tags
goup: github.com/xmlking/logger/logrus: github.com/sirupsen/logrus v1.4.2 is up to date
goup: github.com/xmlking/logger/logrus: github.com/pkg/errors v0.9.1 is up to date
goup: github.com/xmlking/logger/zap: ../ check failed: git: invalid repository: unsecured call to http://../ cancelled: failed to list tags
goup: github.com/xmlking/logger/zap: github.com/pkg/errors v0.9.1 is up to date
goup: github.com/xmlking/logger/zap: go.uber.org/zap check failed: git: invalid repository: unsecured call to http://go.uber.org/zap cancelled: failed to list tags
goup: github.com/xmlking/logger/zerolog: ../ check failed: git: invalid repository: unsecured call to http://../ cancelled: failed to list tags
goup: github.com/xmlking/logger/zerolog: github.com/pkg/errors v0.9.1 is up to date
goup: github.com/xmlking/logger/zerolog: github.com/rs/zerolog v1.18.0 is up to date
```

**Comment by rvflash** [comment]
Hi Sumanth,

Thank you for this feedback, I look at this to fix it as soon as possible.

**Comment by rvflash** [comment]
Hi, now fixed with the last release: v0.2.0.
